### PR TITLE
add secp384r1 mapping

### PR DESCRIPTION
**Pull request by PatStLouis**
I'm leveraging this library to evaluate `deviceResponse` objects from third party vendors. This was a required addition, could I get a review on this?

### CODE REVIEW

File: pymdoccbor/settings.py [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
 
 COSEKEY_HAZMAT_CRV_MAP = {
-    "secp256r1": "P_256"
+    "secp256r1": "P_256",
+    "secp384r1": "P_384"
 }
 
 CRV_LEN_MAP = {
     "secp256r1": 32,
+    "secp384r1": 48,
```
Comment (PascalDR) [inline pymdoccbor/settings.py:11]:
```suggestion
    "secp384r1": 48,
    "secp521r1": 66
```

**Review by PascalDR — CHANGES_REQUESTED** [review]
Is better if we add secp521 too to support all cases of nist curves implemented in pycose.

For reference: [curves.py](https://github.com/TimothyClaeys/pycose/blob/master/pycose/keys/curves.py)

**Review by PatStLouis — COMMENTED** [review]
LGTM!

**Comment by PatStLouis** [comment]
@PascalDR I committed the change request, are we good to request a merge?

**Review by peppelinux — APPROVED** [review]

File: pymdoccbor/settings.py [hunk 2]
```diff
@@ -2,11 +2,15 @@
 import os
 
 COSEKEY_HAZMAT_CRV_MAP = {
-    "secp256r1": "P_256"
+    "secp256r1": "P_256",
+    "secp384r1": "P_384",
+    "secp521r1":  "P_521"
```
Comment (peppelinux) [inline pymdoccbor/settings.py:7]:
```suggestion
    "secp521r1": "P_521"
```

**Comment by peppelinux** [comment]
@PatStLouis could I ask you to set 0.7.0 here?
https://github.com/IdentityPython/pyMDOC-CBOR/blob/main/pymdoccbor/__init__.py
this would allow us to create a new release

**Comment by PatStLouis** [comment]
@peppelinux done!

**Pull request merged**